sdk/eventgrid/azure-messaging-eventgrid/swagger/README.md

```diff
@@ -68,6 +68,14 @@ directive:
     - rename-model:
         from: ResourceWriteSuccessData
         to: ResourceWriteSuccessEventData
+    - rename-model:
+        from: RedisImportRDBCompletedEventData
+        to: RedisImportRdbCompletedEventData
+    - rename-model:
+        from: RedisExportRdbCompletedEventData
+        to: RedisExportRDBCompletedEventData        
```

Comment:
Looks like these are inverted. `from` should have `*RDB*` and `to` should have `*Rdb*`.

Reply:
reversed.